diff --git a/kimia_infer/api/kimia.py b/kimia_infer/api/kimia.py
index 622b92d..e3b2dd6 100644
--- a/kimia_infer/api/kimia.py
+++ b/kimia_infer/api/kimia.py
@@ -6,6 +6,5 @@ from huggingface_hub import cached_assets_path
 from transformers import AutoModelForCausalLM
 
-from kimia_infer.models.detokenizer import get_audio_detokenizer
 from .prompt_manager import KimiAPromptManager
 from kimia_infer.utils.sampler import KimiASampler
 from huggingface_hub import snapshot_download
@@ -40,6 +39,8 @@ class KimiAudio(object):
 
         if load_detokenizer:
             logger.info(f"Loading detokenizer")
+            from kimia_infer.models.detokenizer import get_audio_detokenizer
+
             # need to compile extension moudules for the first time, it may take several minutes.
             self.detokenizer = get_audio_detokenizer(cache_path)
         else:
